features/step_definitions/cookies_steps.rb

```diff
@@ -0,0 +1,24 @@
+When('I see the cookies banner') do
+  expect(page).to have_css '.govuk-cookie-banner'
+  expect(page.find('.govuk-cookie-banner').visible?).to eq true
+  expect(page).to have_content I18n.t('layouts.cookie_banner.accept_button')
```

Comment:
prefer to copy the actual english text, rather then call the translation the view uses. This then tests the content.